exasol/ds/sandbox/cli/commands/create_docker_image.py

```diff
@@ -0,0 +1,32 @@
+import click
+
+from exasol.ds.sandbox.cli.cli import cli
+from exasol.ds.sandbox.cli.common import add_options
+from exasol.ds.sandbox.lib.dss_docker import DssDockerImage
+
+
+_options = [
+    click.option(
+        '--repository', type=str, metavar="ORG/REPO", show_default=True,
+        default="exasol/data-science-sandbox",
+        help="Organization and repository on hub.docker.com to publish the docker image to."),
+    click.option(
+        '--publish', type=bool, default=False, show_default=True,
+        help="Whether to publish the created Docker image"),
+]
+
+
+@cli.command()
+@add_options(_options)
+def create_docker_image(repository: str, publish: bool):
```

Comment:
How do you define the tag? The default tag should be the project version, but for testing it is probably a good idea to be able to specify the tag.

Reply:
I think also the log level is missing